core_pb/src/driving/motors.rs

```diff
@@ -114,11 +116,15 @@ pub async fn motors_task<T: RobotMotorsBehavior>(
                     // calculate pid
                     data.pid_controllers[m].setpoint(data.set_points[m]);
                     let output = data.pid_controllers[m].next_control_output(measured_speeds[m]);
-                    let output = output.output;
-                    if output > 0.0 {
-                        data.pwm[m] = [output.abs().round() as u16, 0];
+                    let output_built = output.output + data.prev_out[m];
+                    data.prev_out[m] = output_built;
+                    println!("motor {:?} output_built={:?}, input={:?}, speed={:?}", m, output_built, data.set_points[m], measured_speeds[m]);
```

Comment:
pls change to debug!, should be able to import from use crate::driving::debug because it has a cfg to swap between std and nostd environments

Reply:
was removed before I posted this

Reply:
I'll just get rid of it. Not relevant anymore